### DISCUSSION TITLE
Possible to do without soldering?

### DISCUSSION THREAD

**Discussion by frank-m** [Zer0-bit/gaggiuino#140] [Q&A] [answered]
Is there any chance of doing this mod without soldering?

I have no experience soldering :)

**Comment by Zer0-bit** [comment, marked as answer]
Hi - you'll just do yourself  a big favour if you solder everything properly so all components have proper electrical conductivity, there's just too many issues you might bump into without soldering and troubleshooting will be a nightmare, i would suggest starting with some soldering basics to familiarise yourself how easy it is to solder: https://www.youtube.com/watch?v=6rmErwU5E-k
